### PR TITLE
Add Block Copy updates and Spellcheck

### PR DESCRIPTION
**Pull request by casmwenger**
# Description

Please include a summary of the change and which issue is fixed or implemented. Please also include relevant motivation and context (e.g. links, docs, tickets etc.).

**This pull request updates spelling errors and adds consistency to the writing style for the Workflows Add Block side panel following these guidelines: https://docs.google.com/presentation/d/1xBQH7HGS9SEhEpEWUCRWiXOqFg6R4DgyZhM_r4zK5Is/edit?usp=sharing.** 

## Type of change

Please delete options that are not relevant.

-   [X] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] This change requires a documentation update

## How has this change been tested, please provide a testcase or example of how you tested the change?

Tested locally and reviewed copy in separate doc to make sure it didn't contain any spelling errors. 

## Any specific deployment considerations

For example, documentation changes, usability, usage/costs, secrets, etc.

## Docs

-   [ ] Docs updated? What were the changes:

### CODE REVIEW

File: inference/core/workflows/core_steps/analytics/time_in_zone/v1.py [hunk 1]
```diff
@@ -30,7 +30,7 @@
 )
 
 OUTPUT_KEY: str = "timed_detections"
-SHORT_DESCRIPTION = "Track duration of time spent by objects in zone"
+SHORT_DESCRIPTION = "Track duration of time spent by objects in zone."
```
Comment (EmilyGavrilenko) [inline inference/core/workflows/core_steps/analytics/time_in_zone/v1.py:33]: Track how long objects spend in a zone.

File: inference/core/workflows/core_steps/analytics/time_in_zone/v2.py [hunk 2]
```diff
@@ -28,7 +28,7 @@
 )
 
 OUTPUT_KEY: str = "timed_detections"
-SHORT_DESCRIPTION = "Track duration of time spent by objects in zone"
+SHORT_DESCRIPTION = "Track duration of time spent by objects in zone."
```
Comment (EmilyGavrilenko) [inline inference/core/workflows/core_steps/analytics/time_in_zone/v2.py:31]: Track how long objects spend in a zone.

File: inference/core/workflows/core_steps/classical_cv/camera_focus/v1.py [hunk 3]
```diff
@@ -22,7 +22,7 @@
     WorkflowBlockManifest,
 )
 
-SHORT_DESCRIPTION: str = "Helps focus a camera by providing a focus measure."
+SHORT_DESCRIPTION: str = "Focus a camera by providing a focus measure."
```
Comment (EmilyGavrilenko) [inline inference/core/workflows/core_steps/classical_cv/camera_focus/v1.py:25]: Calculate a score to indicate how well-focused a camera is.

File: inference/core/workflows/core_steps/formatters/first_non_empty_or_default/v1.py [hunk 4]
```diff
@@ -18,7 +18,7 @@
 fills with default value to make it compliant with further processing.
 """
 
-SHORT_DESCRIPTION = "Takes first non-empty data element or default"
+SHORT_DESCRIPTION = "Take the first non-empty data element or default."
```
Comment (EmilyGavrilenko) [inline inference/core/workflows/core_steps/formatters/first_non_empty_or_default/v1.py:21]: Take the first non-empty data element or the configured default value.

File: inference/core/workflows/core_steps/sinks/local_file/v1.py [hunk 5]
```diff
@@ -69,7 +69,7 @@ class BlockManifest(WorkflowBlockManifest):
         json_schema_extra={
             "name": "Local File Sink",
             "version": "v1",
-            "short_description": "Saves data into local file",
+            "short_description": "Save data into a local file.",
```
Comment (EmilyGavrilenko) [inline inference/core/workflows/core_steps/sinks/local_file/v1.py:72]: Save data to a local file.

File: inference/core/workflows/core_steps/transformations/bounding_rect/v1.py [hunk 6]
```diff
@@ -24,7 +24,7 @@
 
 OUTPUT_KEY: str = "detections_with_rect"
 
-SHORT_DESCRIPTION = "Find minimal bounding rectangle surrounding detection contour"
+SHORT_DESCRIPTION = "Find the minimal bounding rectangle surrounding detection contour."
```
Comment (EmilyGavrilenko) [inline inference/core/workflows/core_steps/transformations/bounding_rect/v1.py:27]: Find the minimal bounding box surrounding the detected polygon.

File: inference/core/workflows/core_steps/transformations/bounding_rect/v1.py [hunk 7]
```diff
@@ -24,7 +24,7 @@
...
 LONG_DESCRIPTION = """
 The `BoundingRect` is a transformer block designed to simplify polygon
 to the minimum boundig rectangle.
```
Comment (EmilyGavrilenko) [inline inference/core/workflows/core_steps/transformations/bounding_rect/v1.py:30]: Looks like this has typos

File: inference/core/workflows/core_steps/transformations/image_slicer/v1.py [hunk 8]
```diff
@@ -49,7 +49,7 @@ class BlockManifest(WorkflowBlockManifest):
         json_schema_extra={
             "name": "Image Slicer",
             "version": "v1",
-            "short_description": "Splits input image into series of smaller images to perform accurate prediction.",
+            "short_description": "Consolidate detections from multiple sections of an input image into a single detection.",
```
Comment (EmilyGavrilenko) [inline inference/core/workflows/core_steps/transformations/image_slicer/v1.py:52]: wrong description, should be:

`Tile the input image into a list of smaller images to perform small object detection.`

**Review by PawelPeczek-Roboflow — APPROVED** [review]

**Review by grzegorz-roboflow — APPROVED** [review]

**Pull request merged**